src/stat.c

```diff
@@ -33,6 +33,8 @@
 
 wrapper(stat, int, (const char * file_name, struct stat * buf))
 {
+    char fakechroot_abspath[FAKECHROOT_PATH_MAX];
+    char fakechroot_buf[FAKECHROOT_PATH_MAX];
```

Comment:
What's the point of adding two statically allocated string variables and not using them at all?

Reply:
Nevermind, I looked at the `expand_chroot_path()` macro and it uses these two for the expansion.  A non-obvious approach, but I guess we work with what we've got :)